tests/vmi_iothreads_test.go

```diff
@@ -75,5 +79,148 @@ var _ = Describe("IOThreads", func() {
 			Expect(len(newVMI.Spec.Domain.Devices.Disks)).To(Equal(1))
 		})
 
+		It("[test_id:864][ref_id:2065] Should honor a mix of shared and dedicated ioThreadsPolicy", func() {
+			policy := v1.IOThreadsPolicyShared
+			vmi.Spec.Domain.IOThreadsPolicy = &policy
+
+			// The disk that came with the VMI
+			dedicated := true
+			vmi.Spec.Domain.Devices.Disks[0].DedicatedIOThread = &dedicated
```

Comment:
What's a point declare same variable here and in the 137 lines in the different contexts? Maybe better to declare it globally? And set it in the test cases how it requires?

Reply:
fixed thanks!